debian/changelog

```diff
@@ -0,0 +1,5 @@
+python-securesystemslib (0.10.8) unstable; urgency=low
+
+  * Initial Debian release.
+
+ -- Saintiago Torres-Arias <santiago@nyu.edu>  Thu, 02 Oct 2014 13:52:55 +0200
```

Comment:
Why is this timestamp set to 2014?  What is it supposed to signify?

Reply:
I should fix this, the timestamp should be set for when we intend to merge the `debian` folder or so. I can also match it with the tag release for `0.10.8`